packages/ember-htmlbars/tests/integration/select_in_template_test.js

```diff
@@ -152,35 +152,39 @@ QUnit.test('works from a template', function() {
   equal(select.get('selection'), erik, 'Selection was maintained after new option was added');
 });
 
-QUnit.test('upon content change, the DOM should reflect the selection (#481)', function() {
-  var userOne = { name: 'Mike', options: Ember.A(['a', 'b']), selectedOption: 'a' };
-  var userTwo = { name: 'John', options: Ember.A(['c', 'd']), selectedOption: 'd' };
-
-  view = EmberView.create({
-    user: userOne,
-    selectView: SelectView,
-    template: compile(
-      '{{view view.selectView viewName="select"' +
-      '    content=view.user.options' +
-      '    selection=view.user.selectedOption}}'
-    )
+if (window.navigator.userAgent.indexOf('Firefox') === -1) {
```

Comment:
Can you use the feature detect that exists in the `ember-metal/environment` module [here](https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/environment.js#L28) instead?